rollup.config.js

```diff
@@ -3,22 +3,47 @@ const babel = require('@rollup/plugin-babel').babel;
 const nodeResolve = require('@rollup/plugin-node-resolve').nodeResolve;
 const replace = require('@rollup/plugin-replace');
 const { terser } = require('rollup-plugin-terser');
+const pkg = require('./package.json');
 
 const env = process.env.NODE_ENV;
-const isProd = env === 'production';
 const version = process.env.npm_package_version;
-const license = fs.readFileSync('./src/license_header', 'utf8').replace(/VERSION/ig, version);
+const license = fs
+  .readFileSync('./src/license_header', 'utf8')
+  .replace(/VERSION/gi, version);
+
+const commonOutputConfig = {
+  name: 'DOMPurify',
+  sourcemap: true,
+  banner: license,
+  exports: 'default',
+};
 
 const config = {
   input: 'src/purify.js',
   external: [],
-  output: {
-    name: 'DOMPurify',
-    globals: {},
-    format: 'umd',
-    sourcemap: true,
-    banner: license,
-  },
+  output: [
+    {
+      ...commonOutputConfig,
+      file: pkg.browser,
+      format: 'umd',
+    },
+    {
+      ...commonOutputConfig,
+      file: pkg.production,
+      format: 'umd',
+      plugins: [terser()],
+    },
+    {
+      ...commonOutputConfig,
+      file: pkg.module,
+      format: 'es',
+    },
+    {
+      ...commonOutputConfig,
+      file: pkg.main,
+      format: 'cjs',
+    },
+  ],
```

Comment:
By setting output to an array, you can set each format and file path.

Also, we can set the plugin individually, so we don't need logic like checking `isProd` to add a `terser`.